03/app.js

```diff
@@ -6,32 +6,57 @@ const root = createRoot(document.querySelector('#root'));
 class Article extends React.Component {
     state = {
         comments: [],
+        content: '',
     }
+
+    handleInputChange = (e) => {
+        this.setState({
+            content: e.target.value,
+        });
+    };
+
+    handleSubmit = (e) => {
+        e.preventDefault();
+        const { content, comments } = this.state;
+
+        if (content.trim()) {
```

Comment:
Zwróć uwagę, że tutaj robisz "trim" - czyli zwracasz ciąg znaków bez białych znaków, ale potem wykorzystujesz ten z nimi. Należałoby w obu przypadkach to zrobić tj. do zmiennej przypisać zmieniony ciąg znaków i go wykorzystać w `if` oraz `state`